treeherder/perf/auto_perf_sheriffing/backfill_reports.py

```diff
@@ -80,7 +81,34 @@ def _ensure_alerts_variety(self, sorted_alerts: List[PerformanceAlert]):
             : self.max_improvements if improvements_only else self.max_alerts
         ]
 
-    def _ensure_platform_variety(self, sorted_all_alerts: List[PerformanceAlert]):
+    def _ensure_distinct_jobs(self, alerts: List[PerformanceAlert]) -> List[PerformanceAlert]:
+        def initial_culprit_job(alert):
+            return alert.initial_culprit_job
```

Comment:
What does `initial_culprit_job` mean?

Reply:
It is the job for the initial data point from that alert